packages/react-composites/tests/browser/call/hermetic/fixture.ts

```diff
@@ -102,7 +102,7 @@ export function defaultMockCallAdapterState(participants?: MockRemoteParticipant
  *
  * Use this to add participants to state created via {@link defaultCallAdapterState}.
  */
-export function defaultMockRemoteParticipant(displayName: string): MockRemoteParticipantState {
+export function defaultMockRemoteParticipant(displayName?: string): MockRemoteParticipantState {
   return {
     identifier: { kind: 'communicationUser', communicationUserId: `8:acs:${displayName}-id` },
```

Comment:
if we don't supply a displayName will the identifier be 8:acs-id?

Perhaps we need to detach the concept of acs user id from display name?

Reply:
Correct. Doesn't really matter. it's just a mock name,